sdk/storage/perf-test/pom.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-storage-blob-perfstress</artifactId>
+  <version>1.0-SNAPSHOT</version>
+
+  <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
Are all the perf tests pinned to JDK 8 or can we have two profiles to run these tests for both JDK 8 and 11?

Reply:
I don't see any issues of having two profiles.
Code is compatible with Java 8 and java 11.
Will add the setup for Java 8 and Java 11 profiles in the Phase 2 PR.

Reply:
Ideally we should be able to run perf tests on all supported JDK versions.  My Java expertise is limited so if anything looks wrong in the `pom.xml` files it probably is wrong :).

Reply:
@mikeharder the pom.xml file looks correct, just wanted to ensure that we can support both JDK 8 and 11. We can add two profiles like @g2vinay said to support both.